src/postgres/src/test/regress/expected/yb_ysql_dump.out

```diff
@@ -38,7 +38,7 @@ ALTER TABLE public.chat_user OWNER TO yugabyte_test;
 CREATE TABLE public.rls_private (
     k integer NOT NULL,
     v text,
-    PRIMARY KEY((k) HASH)
+    CONSTRAINT rls_private_pkey PRIMARY KEY((k) HASH)
```

Comment:
Please ensure to add constraint for `chatID` on line 28 above as well eg.`CONSTRAINT chat_user_pkey PRIMARY KEY(("chatID") HASH)`

Reply:
Done!